### PR TITLE
BTAT-11189 Added Welsh to charge types

### PR DESCRIPTION
**Pull request by lz1kwn**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -84,11 +84,28 @@ payment.noPayment = No payments due right now
 payment.nextPayment = Next payment due
 payment.nextPayments = Payments due
 payment.check = Check what you owe
+payment.afterSubmitted = After you have submitted a return, it can take 24 hours for what you owe to show here.
```
Comment (lewishay) [inline conf/messages:87]: Why have new English messages been added? Can all of the changes to this file be reverted please.

File: conf/messages.cy [hunk 2]
```diff
@@ -84,11 +84,28 @@ payment.noPayment = Nid oes taliadau’n ddyledus nawr
 payment.nextPayment = Taliad nesaf yn ddyledus
 payment.nextPayments = Taliadau sy’n ddyledus
 payment.check = Gwirio’r hyn sydd arnoch
+payment.afterSubmitted = Ar ôl i chi gyflwyno Ffurflen TAW, gall gymryd 24 awr i’r hyn sydd arnoch gael ei ddangos yma.
```
Comment (lewishay) [inline conf/messages.cy:87]: Can you remove all of the new message keys that have been added in this file - we're trying to fill in the gaps with the `???` messages, not add new message keys.

File: conf/messages.cy [hunk 3]
```diff
@@ -120,7 +137,7 @@ whatYouOwe.details.agent.paymentHelpOne = Help gyda thalu
 whatYouOwe.details.agent.paymentHelpTwo = Os na all eich cleient dalu’r hyn sydd arno, mae’n bosibl y bydd yn gallu
 whatYouOwe.details.agent.paymentHelpThree = trefnu cynllun talu (yn agor tab newydd)
 whatYouOwe.details.agent.paymentHelpFour = er mwyn talu fesul rhandaliad misol.
-whatYouOwe.estimate = ???
+whatYouOwe.estimate = amcangyfrif
```
Comment (lewishay) [inline conf/messages.cy:140]: Can you revert all of the changes from here up until line 235 please - these are messages for the breakdown screens and should not be done in your task as they have already been addressed in BTAT-11190

File: conf/messages.cy [hunk 4]
```diff
@@ -459,59 +476,59 @@ chargeType.refundDescription = gan i chi ofyn am ad-daliad o ordaliad a wnaethoc
...
 
 chargeType.POAInstalmentTitle = Rhandaliad taliad ar gyfrif
-chargeType.POAInstalLPITitle = ???
+chargeType.POAInstalLPITitle = Llog ar asesiad i ddiogelu
 
 chargeType.POAReturnDebitChargeTitle = Balans taliad ar gyfrif
 chargeType.POAReturnCreditChargeTitle = Ad-daliad taliad ar gyfrif
 
-chargeType.VatReturnPOALPITitle = ???
+chargeType.VatReturnPOALPITitle = Llog ar daliad ar gyfrif
```
Comment (lewishay) [inline conf/messages.cy:484]: You haven't added any LPP translations but I can see we do have at least some of them translated on the spreadsheets. The spreadsheets have them translated as a full sentence with the "for the period...", which is not a direct match for the LPP titles, but by comparing a few of the translations you can trim out the "for the period..." text which leaves what we want.

**Review by lewishay — CHANGES_REQUESTED** [review]

**Comment by lewishay** [comment]
SBT has been upgraded to 1.7.2 in PR for BTAT-11190 so it can be ignored here

File: conf/messages.cy (hunk 2 above)
Comment (lz1kwn) [reply conf/messages.cy:87]: Removed new messages and replaced with ???

File: conf/messages.cy [hunk 5]
```diff
@@ -464,54 +464,54 @@ chargeType.POAInstalLPITitle = ???
...
-chargeType.VATPA2ndLPPLPITitle = ???
-chargeType.VATPA1stLPPTitle = ???
-chargeType.VATPA2ndLPPTitle = ???
-
-chargeType.VATAdditionalAssessmentLPITitle = ???
-chargeType.VATAA1stLPPTitle = ???
-chargeType.VATAA2ndLPPTitle = ???
-chargeType.VATAA1stLPPLPITitle = ???
-chargeType.VATAA2ndLPPLPITitle = ???
+chargeType.VatReturnPOALPITitle = Llog ar daliad ar gyfrif
```
Comment (lewishay) [inline conf/messages.cy:467]: Can you please add in the Welsh for "Interest on protective assessment", line 451 in this file, I can see this translation in the BTAT-10582 spreadsheet

Comment (lewishay) [inline conf/messages.cy:467]: Can you please add in the Welsh for the error correction charges, lines 426 to 430 in this file, I can see these translations in the BTAT-10582 spreadsheet

Comment (lewishay) [inline conf/messages.cy:467]: Can you please add in the Welsh for the LPP headings, lines 302 & 303 in this file, I can see these translations in the BTAT-10582 spreadsheet

**Review by lewishay — CHANGES_REQUESTED** [review]

File: conf/messages.cy (hunk 5 above)
Comment (lz1kwn) [reply conf/messages.cy:467]: Added Welsh

Comment (lz1kwn) [reply conf/messages.cy:467]: Added welsh to lines 426 and 430

Comment (lz1kwn) [reply conf/messages.cy:467]: Added welsh

File: conf/messages.cy [hunk 6]
```diff
@@ -423,11 +423,11 @@ chargeType.vatPADefaultInterestDescription = a godir ar yr asesiad i ddiogelu
 chargeType.vatErrorCorrectionNoticeDefaultInterestTitle = Llog diffygdalu ar gywiro gwall
 chargeType.vatErrorCorrectionNoticeDefaultInterestDescription = a godir ar y swm a aseswyd
 
-chargeType.vatErrorCorrectionLPITitle = ???
+chargeType.vatErrorCorrectionLPITitle = Llog ar gywiriad gwall
```
[the remark this answers is not in the document]
Comment (lz1kwn) [reply conf/messages.cy:426]: added welsh for error correction

**Pull request merged**